### PR TITLE
Warning on build if local.yaml doesn't exist

### PR DESCRIPTION
**Pull request by ccjjmartin**
## Purpose:
- Warning on build if local.yaml doesn't exist

## Note to reviewer:
- I would love to see any other uses for the local.yml file added to this PR.  I am curious how other developers are using it.  Attaching to a local backend was the only use case I could see right now.

## Reproduce warning:
- Get a fresh copy of scrummy
- Run `yarn`
- Run `npm run build`
- Notice this warning:
```
WARNING in ./config/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./local.yaml in /Users/developer/repos/contrib/scrummy-react-dom/config
 @ ./config/index.js 9:16-39
```

## Notes:
- It appears that someone has looked into this before because it is enclosed in a try catch
- I looked into using the fs module to check if the file exists with no luck.  Apparently webkit doesn't like the fs module and this might be because this is eventually sent into someone's browser.
- After much trial and error with the fs module I decided it was easier to just commit a blank local.yaml file.  Found out that changes will attempt to be committed if anyone makes any so I removed the file.
- Decided updating the docs was a better approach here.
- Didn't realize that the README.md was auto generated in my last PR so I fixed my mistake of directly editing the README.md and used emdaer this time around.

## Testing:
- Follow the reproduce bug steps but notice the error is gone.

### CODE REVIEW

File: docs/backend.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Local Backend Server Setup:
+
+### Note:
+* The default is to connect to the Herokuapp at ws://scrummy-server.herokuapp.com.
```
Comment (infiniteluke) [inline docs/backend.md]: Can we have this read: "The deployed application connects to ws://scrummy-server.herokuapp.com."

File: docs/backend.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+### Scrummy Server:
+* Copy `config/example.local.yaml` to `config/local.yaml` to connect to the [Scrummy Server](https://github.com/fourkitchens/scrummy-server).
+* In the Scrummy Server repo change `config/default.yml` to have a port line that reads `port: 8081`.
+* Open two terminal windows and start up both dev servers with `npm run start` in the projects root directories.
```
Comment (infiniteluke) [inline docs/backend.md]: "...start up both apps..."

sp "projects'"

**Review by infiniteluke — CHANGES_REQUESTED** [review]
This is great work! A couple of wording nits in the readme. Thanks!

**Comment by infiniteluke** [comment]
To answer your question about local config, I only have `apiUrl: ws://localhost:8001` in my `local.yml` 👍

**Comment by ccjjmartin** [comment]
@infiniteluke Thanks for the quick reviews on these.  I updated the docs for your comments except the one about sp "projects".  I haven't seen the abbreviation sp before.  I am guessing that is spelling error.  If so, I went with the plural of "projects" to refer to both projects rather than the possessive "project's" which would seem to only refer to one.  I can update this to "project's" if you prefer though.  Let me know if sp means something else.

**Comment by infiniteluke** [comment]
Hey @ccjjmartin - Yes, sp was a bit cryptic, but you're right, I meant spelling error. In this case we'll still want to use `projects'` since it is possessive of their directories and there are multiple of them (server/client).

**Review by infiniteluke — APPROVED** [review]

**Pull request merged**